### DISCUSSION TITLE
iptables in new Security Onion Installation

### DISCUSSION THREAD

**Discussion by jenbate** [Security-Onion-Solutions/securityonion#11924] [General]
I am new at SO and trying to set it up in my home lab. 

I have Security Onion 2.4.30-20231121 in a VM installed on an ESXi 8.0. 
I used 4 CPUs, 12 GB Ram, and 200 GB memory.
I have installed everything and it seems okay, the so-status says that all the containers her fine. 

But I was not getting any alerts. Meanwhile in the command line of SO I saw a ton of iptables alerts.  Endless. I finally shut off iptables and the chatter stopped, and in the SO GUI I got a few alerts. Hooray. But after a few minutes iptables turns back on. 

Any ideas? Thanks I have no idea if this is a new issue or not. I have searched the forums a bunch.  THANK YOU

**Comment by jenbate** [comment]
solved it never mind!

**Reply by conmank3** [reply]
What did you do to resolve it if you don't mind me asking? We are doing fresh installs and getting hammered with IPTables-dropped before login. Thank you!
